CHANGELOG.md

```diff
@@ -3,6 +3,83 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.13.1
+
+_Aug 22, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Allow [filtering the axis on zoom](https://mui.com/x/react-charts/zoom-and-pan/#zoom-filtering), making the axis adapt by removing values outside the view.
+
+  <img width="600" src="https://github.com/user-attachments/assets/e65bbd00-d2a8-4136-81cd-3598f1373c16" alt="filtering the axis on zoom" />
+
+- 📊 Improve bar chart performances
+- 🌍 Improve Czech (cs-CZ) locale Data Grid
+- 🌍 Improve Chinese (zh-HK) locale on the Date and Time Pickers
+- 🌍 Improve Vietnamese (vi-VN) locale on the Date and Time Pickers
+- 🐞 Bugfixes
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.13.1`
+
+- [data grid] Use readonly array result for getTreeDataPath (#11743) @pcorpet
+- [data grid] use `event.key` for `Tab` and `Escape` keys (#14170) @k-rajat19
+- [DataGrid] include `api` in `gridCellParams` interface (#14201) @k-rajat19
+- [l10n] Improve Czech (cs-CZ) locale (#14135) @chirimiri22
+
+#### `@mui/x-data-grid-pro@7.13.1` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@7.13.1`.
+
+#### `@mui/x-data-grid-premium@7.13.1` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@7.13.1`, plus:
+
+- [DataGridPremium] Clipboard paste not working for a single cell if the current page is not the first page (#14261) @arminmeh
+- [DataGridPremium] Fix `onCellSelectionModelChange` not triggered when additional cell range is selected (#14199) @arminmeh
+
+### Date and Time Pickers
+
+#### `@mui/x-date-pickers@7.13.1`
+
+- [l10n] Improve Chinese (zh-HK) locale (#13289) @yeeharn
+- [l10n] Improve Vietnamese (vi-VN) locale (#14238) @locnbk2002
+
+#### `@mui/x-date-pickers-pro@7.13.1` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@7.13.1`.
+
+### Charts
+
+#### `@mui/x-charts@7.13.1`
+
+- [charts] Fix grid overflow with zooming (#14280) @alexfauquette
+- [charts] Improve bar chart performances (#14278) @alexfauquette
+- [charts] Test pointer events (#14042) @alexfauquette
+
+#### `@mui/x-date-charts-pro@7.0.0-alpha.2` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-charts@7.13.1`, plus:
+
+- [charts-pro] Zoom axis filtering (#14121) @JCQuintas
+
+### Docs
+
+- [docs] Consistent use of UTC and timezones (#14250) @oliviertassinari
+- [docs] Fix missing leading slashes in URLs (#14249) @oliviertassinari
+- [docs][pickers] Dash usage revision (#14260) @arthurbalduini
```

Comment:
AFAIK, we haven't yet used this standard that Core is using in the changelog. 🤔 
Not sure if we should start doing it in certain cases...
What do others think?
I would even consider a counter argument—keeping only the second prefix in such cases as we are already listing changes in the `Docs` section. 🙈 🤷

Reply:
I agree that we could omit of the [docs] label since it's redundant with the section title.